sound/soc/sof/topology.c

```diff
@@ -359,6 +359,14 @@ int get_token_uuid(void *elem, void *object, u32 offset)
 	return 0;
 }
 
+int get_token_string(void *elem, void *object, u32 offset)
+{
+	u8 *dst = (u8 *)object + offset;
+
+	memcpy(dst, elem, SNDRV_CTL_ELEM_ID_NAME_MAXLEN);
```

Comment:
This function can only be used for getting name because of SNDRV_CTL_ELEM_ID_NAME_MAXLEN. How about renaming it to get_token_name?

Reply:
This function is used to get the string from
```
struct snd_soc_tplg_vendor_string_elem {
	__le32 token;
	char string[SNDRV_CTL_ELEM_ID_NAME_MAXLEN];
} __attribute__((packed));
```
I'd like to align with alsa here, which is also aligned with the naming conversion with the get_token_uuid/get_token_u32/get_token_u16 variants.